contrib/camel/config/500-controller.yaml

```diff
@@ -32,12 +32,5 @@ spec:
       containers:
       - image: github.com/knative/eventing-contrib/contrib/camel/cmd/controller
         name: manager
-        resources:
-          limits:
-            cpu: 100m
-            memory: 30Mi
-          requests:
-            cpu: 20m
-            memory: 20Mi
```

Comment:
You might want to leave `requests` (which is a minimum guarantee), but remove `limits` (which is a hard maximum).

Reply:
Thanks @evankanderson, it seems to work well with only requests. I've added them back :+1: